docs/reference/nuctl/nuctl.md

```diff
@@ -1,48 +1,63 @@
-# Nuctl
+# The Nuclio CLI (nuctl)
 
 #### In this document
 
-- [About](#about)
-- [Download](#download)
+- [Overview](#overview)
+- [Download and Installation](#download-n-install)
 - [Usage](#usage)
-- [Running platform](#running-platform)
-- [Docker](#docker)
-- [Kubernetes](#kubernetes)
+- [The running platform](#running-platform)
+  - [Docker](#docker)
+  - [Kubernetes](#kubernetes)
 
-## About
+<a id="overview"></a>
+## Overview
 
-Nuctl is Nuclio's command-line interface that provides you with all the nuclio features, accessible from your terminal
+`nuctl` is Nuclio's command-line interface (CLI), which provides command-line access to Nuclio features.
 
-## Download
+<a id="download-n-install"></a>
+## Download and Installation
 
-To install Nuctl all you need to do, is simply visit Nuclio [releases page](https://github.com/nuclio/nuclio/releases)
-and download the binary appropriate to your platform (e.g.: `darwin` if you have `macOS`)
+To install `nuctl`, simply visit the Nuclio [releases page](https://github.com/nuclio/nuclio/releases) and download the appropriate CLI binary for your environment (for example, **nuctl-&lt;version&gt;-darwin-amd64** for a machine running macOS).
 
-You can use the following bash snippet to download the latest nuctl release
-```bash
+You can use the following command to download the latest `nuctl` release:
+```sh
 curl -s https://api.github.com/repos/nuclio/nuclio/releases/latest \
 			| grep -i "browser_download_url.*nuctl.*$(uname)" \
 			| cut -d : -f 2,3 \
 			| tr -d \" \
 			| wget -O nuctl -qi - && chmod +x nuctl
 ```
 
+<a id="usage"></a>
 ## Usage
 
-Once downloaded, an informative help section is available using `nuctl --help`
+After you download `nuctl`, use the `--help` option to see full usage instructions:
+```sh
+nuctl --help
+```
 
-## Running platform
+<a id="running-platform"></a>
+## The running platform
 
-Nuctl will automatically identify its running platform, whether it is Docker or Kubernetes.
+`nuctl` automatically identifies the platform from which it's being run.
+You can also use the `--platform` CLI flag to ensure that you're running against a specific platform, as explained in this reference:
 
-To ensure you run Nuctl against a specific platform, use `--platform kube` for Kubernetes or `--platform local` for docker
+- [Docker](#docker)
+- [Kubernetes](#kubernetes)
 
+<a id="docker"></a>
 ### Docker
```

Comment:
```suggestion
### Docker (local)
```
Since it's called "local" in the cli arg, I believe it should be made clear it's a synonym to docker in this context

Reply:
@omesser I would prefer "Docker (local registry)" or "Local Docker registry" and I think that based on your comment we should also replace "Docker" instances in the content  with "a local Docker registry". Please confirm.

Reply:
@Sharon-iguazio - We're talking about docker runtime vs k8s runtime - this has nothing to do with the docker-registry nuclio will work against. The word registry should not appear here

Reply:
@omesser I changed "Docker" to "Local Docker" — commit 40b4611.